web/app.js

```diff
@@ -176,6 +176,7 @@ var PDFViewerApplication = {
   preferencePdfBugEnabled: false,
   preferenceShowPreviousViewOnLoad: true,
   preferenceDefaultZoomValue: '',
+  preferenceEnhanceTextSelection: true,
```

Comment:
> By the way, if this is enabled by default, do we still need separate enhance tests?

I'd definitely say that we need the separate tests, since this PR doesn't expand the divs until you click the page, and without these specific tests we have no way of ensuring that we don't accidentally regress that mode.